Web Quiz Engine (Kotlin)/task/src/engine/service/QuizService.kt

```diff
@@ -24,6 +28,21 @@ class QuizService(
             )
         }
 
+    fun deleteQuizById(quizId: Int): ResponseEntity<Unit> =
```

Comment:
Instead of making this method return type `ResponseEntity<Unit>`, just make it `Unit` or no return type.

You only need to return ResponseEntities at the Service level if there is logic within the Service method that will impact the Http response status.
It is not the case for this method.

Reply:
Ok yes I paused on that one. Makes sense what you say.
Done.